framework/wazuh/tests/test_manager.py

```diff
@@ -53,7 +72,11 @@ def test_restart_ok(test_manager):
     ('input_lists_file', 'output_lists_file')
 ])
 @patch('wazuh.common.ossec_path', test_data_path)
-def test_upload_file(test_manager, input_file, output_file):
+@patch('time.time', lambda: 0)
+@patch('random.randint', lambda x, y: 0)
+@patch('wazuh.manager.chmod')
+@patch('wazuh.manager.move')
+def test_upload_file(move_mock, chmod_mock, test_manager, input_file, output_file):
```

Comment:
Every patch decorator implies the corresponding parameter in the test definition.

Reply:
Done here: https://github.com/wazuh/wazuh/pull/2718/commits/b4d6d3b6e8919708414a02f09eff8d463b10850c